src/liballoc/rc.rs

```diff
@@ -1179,6 +1179,14 @@ impl<T> From<Vec<T>> for Rc<[T]> {
     }
 }
 
+#[unstable(feature = "rc_into_nonnull", reason = "newly added", issue = "0")]
+impl<T: ?Sized> Into<NonNull<T>> for Rc<T> {
```

Comment:
I would like to add an equivalent inherent function like we have for Box: Rc::into_raw_non_null and Arc::into_raw_non_null.

Reply:
I'll wait for https://github.com/rust-lang/rust/pull/56998#discussion_r246454126 to be resolved first.